package.json

```diff
@@ -21,9 +21,13 @@
     "@angular/platform-browser": "~8.2.14",
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
+    "csv-parser": "^2.3.3",
     "fhirclient": "^2.3.1",
     "fhirpath": "^2.3.0",
     "file-saver": "^2.0.2",
+    "fs": "0.0.1-security",
```

Comment:
`fs` isn't a real dependency. It's a place-holder to prevent people from overwriting the built-in Node `fs` module. It is never needed.